packages/core/src/components/TextField/TextField.scss

```diff
@@ -64,6 +68,14 @@ Style guide: components.text-field
   }
 }
 
+.ds-c-field--tiny {
```

Comment:
We use `small` in other places..should we use that to be consistent? I.e. `ds-c-choice--small`, `ds-c-button--small`

Reply:
Or do we need even more 😮values? `tiny`, `small`, `medium`, `large`, `child-size`

Reply:
🙅‍♂️

Reply:
> `child-size`

If someone doesn't respond with a gif in the next hour, I'll have to

Reply:
![](https://i.imgur.com/FWbmQNx.gif)

Reply:
That's horrifying 😱 I was going more for 
![200w_d](https://user-images.githubusercontent.com/7595652/36993282-43a2856e-2062-11e8-9707-f563e7a9e33f.gif)
 😛

Reply:
Wow, sorry @hannaliebl for vandalizing your PR. The code looks good to me 👍

Reply:
I was going more for this, but couldn't find a version in GIF format. http://nymag.com/selectall/2018/02/baby-steals-food-from-bird-in-viral-gif.html

Sorry, not sorry @hannaliebl

Reply:
That was quite a journey. 😄